packages/cli/src/commonlib/azureLogin.ts

```diff
@@ -164,20 +190,35 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
 
   private async updateLoginStatus(): Promise<void> {
     if (
+      AzureAccountManager.codeFlowInstance !== undefined &&
       AzureAccountManager.codeFlowInstance.account &&
       AzureAccountManager.statusChange !== undefined
     ) {
       const credential = await this.doGetAccountCredentialAsync();
       const accessToken = await credential?.getToken();
       const accountJson = await this.getJsonObject();
       await AzureAccountManager.statusChange("SignedIn", accessToken?.accessToken, accountJson);
+    } else if (
+      AzureAccountManager.codeFlowTenantInstance !== undefined &&
```

Comment:
It seems `if` and `else if` have the same code block here

Reply:
Fixed.